cmd/kubeadm/app/phases/uploadconfig/uploadconfig_test.go

```diff
@@ -50,32 +49,11 @@ func TestUploadConfiguration(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t2 *testing.T) {
-			initialcfg := &kubeadmapiv1beta2.InitConfiguration{
-				LocalAPIEndpoint: kubeadmapiv1beta2.APIEndpoint{
-					AdvertiseAddress: "1.2.3.4",
-				},
-				BootstrapTokens: []kubeadmapiv1beta2.BootstrapToken{
-					{
-						Token: &kubeadmapiv1beta2.BootstrapTokenString{
-							ID:     "abcdef",
-							Secret: "abcdef0123456789",
-						},
-					},
-				},
-				NodeRegistration: kubeadmapiv1beta2.NodeRegistrationOptions{
-					Name:      "node-foo",
-					CRISocket: "/var/run/custom-cri.sock",
```

Comment:
DefaultedStaticInitConfiguration use docker sock.
Need we keep this custom-cri.sock  in the test ?

Reply:
i guess we can apply these custom field values to make the config deviate from the default a little.
will update.

Reply:
updated.